code/stats/medals.h

```diff
@@ -46,6 +47,8 @@ class medal_stuff
 	medal_stuff(const medal_stuff &m);
 	medal_stuff& operator=(const medal_stuff &m);
 
+	const char* get_display_string();
```

Comment:
Medals have `get_display_string` while weapons have `get_display_name`.  This may not matter but I thought I should mention it.